### PR TITLE
oem/oracle: Add some scripts to manage images/instances

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: oem/oracle/get-console-output [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+if [[ -z "${instance_id}" ]]; then
+    echo "Instance ID is required." >&2
+    exit 2
+fi
+
+id=$(bmcs compute console-history capture --instance-id "${instance_id}" | jq -r .data.id)
+trap 'bmcs compute console-history delete --instance-console-history-id "${id}" --force' EXIT
+while true; do
+    state=$(bmcs compute console-history get --instance-console-history-id "${id}" | jq -r '.data["lifecycle-state"]')
+    if [[ "${state}" = SUCCEEDED ]]; then
```
Comment (arithx) [inline oem/oracle/get-console-output]: `SUCCEEDED` isn't wrapped in quotes

File: oem/oracle/get-console-output [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    esac
+done
+
+if [[ -z "${instance_id}" ]]; then
+    echo "Instance ID is required." >&2
+    exit 2
+fi
+
+id=$(bmcs compute console-history capture --instance-id "${instance_id}" | jq -r .data.id)
+trap 'bmcs compute console-history delete --instance-console-history-id "${id}" --force' EXIT
```
Comment (arithx) [inline oem/oracle/get-console-output]: Why are you deleting the console output on exit?

File: oem/oracle/upload-image [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    echo "Display name is required." >&2
+    exit 2
+fi
+
+if [[ -z "${path}" ]]; then
+    echo "Image file is required." >&2
+    exit 2
+fi
+
+object=$(mktemp -u XXXXXXXXXXXXXXXX)
```
Comment (arithx) [inline oem/oracle/upload-image]: no trap to delete this

Comment (bgilbert) [inline oem/oracle/upload-image]: `-u` doesn't create a file; we're just generating a random string here.

File: oem/oracle/get-console-output (hunk 2 above)
Comment (bgilbert) [inline oem/oracle/get-console-output]: There's no point leaving console history objects sitting around in the compartment once we've fetched them.

File: oem/oracle/get-console-output (hunk 1 above)
Comment (bgilbert) [inline oem/oracle/get-console-output]: Constant strings don't need to be.  Sometimes they are as a stylistic thing, but I don't think we're rigid about it.

**Pull request merged**